content/events/t1/repeat/_index.md

```diff
@@ -0,0 +1,8 @@
++++
+template = "events/event_week.html"
+
+title = "Repeat Events"
+
+[extra]
+base_date = "2024-04-15"
```

Comment:
is this necessary? (its long way in the past at the very least)

Reply:
It needs a base date (that is a Monday) for the dates to be relative to (before being scaled foward). That date can be changed but there is no particular reason to (it's that since that was t3 w1 that I copied from iirc)